### PR TITLE
Templating arwiki, bnwiki, cawiki, dewiki, elwiki and eswikiquote

### PR DESCRIPTION
**Pull request by Ladsgroup**
I could go on but I think that's enough for one PR
Also I didn't remove the old parts just to let you test it with the differ

### CODE REVIEW

File: config/wikis/elwiki.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+unbalanced_5k_set_for_review: true
+
+default_sample: 20k_2017
+review_sample: 5k_2017
+
+models:
+    reverted:
+        tuning_params:
+            n_estimators: 500
```
Comment (halfak) [inline config/wikis/elwiki.yaml:23]: WHat happened here?

Comment (halfak) [reply config/wikis/elwiki.yaml:23]: Maybe the other params are getting pulled in as defaults?

Comment (Ladsgroup) [reply config/wikis/elwiki.yaml:23]: Yes: https://github.com/wiki-ai/editquality/blob/master/config/model_defaults.yaml

**Comment by halfak** [comment]
LGTM

**Comment by adamwight** [comment]
Here are the diffs:
https://gist.github.com/adamwight/59fe563993d7b47e4b0cbd908cb52d20

**Comment by Ladsgroup** [comment]
One wiki, bnwiki, is wrong, will fix it tomorrow

**Comment by adamwight** [comment]
Made the adjustments.  Gist diffs are updated to reflect the changes.

**Pull request merged**